src/components/FileInput/index.vue

```diff
@@ -590,6 +590,10 @@ export default Vue.extend({
   flex: 0 0 25%;
   padding: 1rem;
 
+  @media (max-width: 1600px) {
+    flex: 0 0 33.3333333333%;
```

Comment:
maybe two digits after the dot is enough?

Reply:
maybe enough, but I looked in Bootstrap
![image](https://user-images.githubusercontent.com/24324706/105018866-8b8ea200-5a56-11eb-9636-80f4d23c6722.png)